.env.example

```diff
@@ -1,8 +1,7 @@
 DEBUG=True
-DJANGO_SECRET_KEY=3YSxRH/mcqSJpkk40s+0qy+uh+9o5LIBdZtwHOo6oNaLe8+Mx6
```

Comment:
I see you somehow messed up the merge of current master  - can you please do it properly - I don't see any merge commit in you PR.